survey/templates/survey/survey.html

```diff
@@ -186,4 +186,19 @@ <h2>4. Gather Evidence</h2>
                     <h2>5. Make an Improvement Plan</h2>
                 </div>
             </div>
-{% endblock %}
\ No newline at end of file
+
+            <div class="card mb-3">
+                <div class="card-header">
+                    <h2>6. Report and data export</h2>
+                </div>
+                <div class="card-body">
+                    <p>You can generate a printable report of this survey or export survey data to a spreadsheet format here.</p>
+                    <div class="d-flex mt-3 mb-3">
+                        <a href="{% url 'survey_report' survey.id %}" class="btn btn-primary me-3">View report</a>
+                        <a href="{% url 'survey_report' survey.id %}" class="btn btn-primary">Export spreadsheet data</a>
```

Comment:
Once merged with #131, change this to:

```suggestion
                        <a href="{% url 'survey_export' survey.id %}" class="btn btn-primary">Export spreadsheet data</a>
```